sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store/_constants.py

```diff
@@ -0,0 +1,9 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+OFFLINE_STORE_CONNECTION_NAME = "OfflineStoreConnectionName"
+OFFLINE_MATERIALIZATION_STORE_TYPE = "azure_data_lake_gen2"
+OFFLINE_STORE_CONNECTION_CATEGORY = "ADLSGen2"
+DEFAULT_SPARK_RUNTIME_VERSION = "3.1.0"
```

Comment:
Does this align with what was discussed with UX ?

Reply:
this is present in the spec yaml as default.